src/ui/simulator/windows/options/advanced/advanced.cpp

```diff
@@ -941,74 +941,45 @@ void AdvancedParameters::onNumberOfCores(Component::Button&, wxMenu& menu, void*
                  this);
 }
 
-void AdvancedParameters::onSelectNCmin(wxCommandEvent& /* evt */)
+void AdvancedParameters::onSelectNC(Data::NumberOfCoresMode ncMode)
```

Comment:
Not really relevant to your dev, but  **Data::NumberOfCoresMode** isn't a good name, **NumberOfCoresLevel** would be better.
And so **onSelectNC** should be renamed **onSelectNumberOfCoresLevel**